src/stores/state.ts

```diff
@@ -58,6 +58,11 @@ export const useStateStore = defineStore('state', () => {
   const decimalFractionDigits = ref(parseInt(storage.getItem('decimalFractionDigits') ?? '5', 10))
   const showVirtualQwerty = ref(storage.getItem('showVirtualQwerty') === 'true')
   const midiOctaveOffset = ref(parseInt(storage.getItem('midiOctaveOffset') ?? '-1', 10))
+  const showKeyboardCents = ref(storage.getItem('showKeyboardCents') !== 'false')
```

Comment:
These new settings are missing the associated watchers. Might make sense to backport `syncValues`.